rusk-wallet/src/gql.rs

```diff
@@ -58,14 +61,50 @@ struct BlockResponse {
     pub block: Option<Block>,
 }
 
+// See `PhoenixTransactionEvent` for the reason for this struct
+// and allowing dead code here.
 #[serde_as]
-#[derive(Deserialize, Debug)]
-pub struct BlockData {
-    #[serde_as(as = "DisplayFromStr")]
-    pub gas_spent: u64,
-    pub sender: String,
+#[derive(Deserialize, Debug, Clone)]
+#[allow(dead_code)]
+struct Note {
+    stealth_address: StealthAddress,
+}
```

Comment:
Can this be made to work with using the stealth-address directly?

And if this struct is absolutely necessary, please call it `NoteAddress` to reflect its content better.